package.json

```diff
@@ -13,7 +13,7 @@
     "test:tn1ORtn2": "npx cypress-tags run --env \"TAGS=@tagname1 OR @tagname2\"",
     "clean:reports": "if exist cypress\\reports rmdir /S/Q cypress\\reports",
     "pretest": "npm run clean:reports",
-    "scripts": "cypress run --browser chrome",
+    "scripts": "cypress run --browser chrome --headed",
```

Comment:
can we rename this `script name` as `scripts` is too generic a name for scripts? and its parent name is already `scripts` in `package.json`.

Reply:
1. For Chai code block, that changes as we auto format the file. Its same line of code just formatted differently
2. We have changed the scripts to script

Reply:
1. how do you auto-format the codebase? it shouldn't break the line based on the length of the line; looking at the `prettier` configuration here https://github.com/bcgov/sso-requests-e2e/blob/Complete_one_happy_path_E2Etesting/.prettierrc#L4
2. I was saying that the name needs to be more meaningful. what about `cypress-headed`?

Reply:
1. Right Click and Format Document to put everything in line
2. cypress-headed is used to run it via Cypress Runner instead of headless run.

Reply:
hm.. it seems like your default `formatter` for javascript files is not `prettier`, when you right click and click on `Format Document With...`, what options do you see? if you don't see `Prettier - Code formatter`, then you need to install it in VSCode to use it.

Reply:
Yes, it's Javascript - The options I see is Beautify - Typescript and Javascript. I'll try to install Prettier code formatter.

Thanks.

Reply:
1. Updated Formatter to Prettier
2. Updated Scenario for more generic lines